docs/it/metadata_oidc_op.rst

```diff
@@ -129,10 +129,10 @@ L'EC di un OP DEVE configurare un metadata di tipo **"openid_provider"** DEVE co
     - Vedi `OpenID.Discovery#OP_Metadata`_. Vedi signature :ref:`supported_algs`.
     - |spid-icon| |cieid-icon|
   * - **request_object_encryption_alg_values_supported**
-    - Vedi `OpenID.Discovery#OP_Metadata`_. Vedi key encryption :ref:`supported_algs`.
+    - Fino a diversa indicazione di AgID, non deve essere incluso.
```

Comment:
this is not neutral considering CIE

can we rephrase?
can we adopt the section regarding the supported algs even as security/implementation considerations?

Reply:
the claim is supported only by SPID (there is spid-icon only)